.github/workflows/publish.yaml

```diff
@@ -1,9 +1,10 @@
-name: Publish JSON Schema
+name: Publish JSON Schema or stac-model package via Github Release
 on:
   release:
     types: [published]
 jobs:
-  deploy:
+  deploy-schema:
+    if: startsWith(github.ref, 'refs/tags/v') && !startsWith(github.ref, 'refs/tags/stac-model-v')
```

Comment:
I don't think the `&& !startsWith(github.ref, 'refs/tags/stac-model-v')` part is needed? 
I think each tag would have its own trigger, so even if a commit contains both tags, each one should run the pipeline on their own, right?

Reply:
ah true. I'll remove that condition